### DISCUSSION TITLE
Problem with MHCOZY ZG-005-RF

### DISCUSSION THREAD

**Discussion by jnilo1** [Koenkk/zigbee2mqtt#24184] [Devices]
Hello ! 
I tried to install an MHCOZY ZG-005-RF Tuya/Zigbee device through Z2M.
It was recognized as a [MHCOZY TYWB 4ch-RF](https://www.zigbee2mqtt.io/devices/TYWB_4ch-RF.html#mhcozy-tywb-4ch-rf) device.
The MHCOZY ZG-005-RF has 3 states that can be activated by pushing a "mode switch" on the device:
1/ Hold mode: This is the most common mode. The 4 outputs are independent, and pressing a button toggles the corresponding output. If it's in On mode, it switches Off, and vice versa.
2/ Impulse mode: This mode switches the activated output to ON for 1s and then back to OFF. This is used to create an impulse, for example to open a gate or garage door.
3/ Exclusive mode: This mode works like the Maintained world, but only one output can remain ON. As soon as another is activated, the previous one switches back to OFF.
Is it possible to activate those states through Z2M ?
Thank you for your great program & support !

**Comment by imLeeat** [comment]
same problem here. it seems the device sometimes switch itself automatically to impulse mode, and i have to press the mode switch button manually....

**Comment by luispb** [comment]
I have the same issue. Is there a way to change the state using Z2M? And great thank you for support!
